packages/core/src/routes/session.ts

```diff
@@ -1,11 +1,14 @@
 import { LogtoErrorCode } from '@logto/phrases';
+import { UserLogResult, UserLogType } from '@logto/schemas';
 import { conditional } from '@silverhand/essentials';
+import { nanoid } from 'nanoid';
 import { Provider } from 'oidc-provider';
 import { object, string } from 'zod';
 
-import RequestError from '@/errors/RequestError';
+import RequestError, { RequestErrorWithUserLog } from '@/errors/RequestError';
```

Comment:
instead of `RequestErrorWithUserLog` maybe we could use  a more context specific naming convention. i.e. InteractionError(user flow) depends on how we want to categorize the error types globally.

Reply:
Just like above conversation, see LOG-395.